complex_spatial_process.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.                station_df.to_csv(f"fine_area_{i*num_long_splits+j+1}_stations.csv")
</pre><p>can you update this to store the areas of interest under the following file path? </p><p> </p><p><code>data/weather-stations/kelowna</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>